src/main/kotlin/uk/gov/justice/digital/hmpps/supportadditionalneedsapi/service/ALNScreenerService.kt

```diff
@@ -74,6 +74,8 @@ class ALNScreenerService(
         "Detail URL: ${inboundEvent.detailUrl}, reference: ${info.curiousExternalReference}",
     )
 
```

Comment:
The actual fix - check to see if the overall need has changed as a result of this ALN update. if it has then update the schedules as appropriate. 

When we can remove needs in SAN we will need to do a similar thing in that code, I'll create a ticket and reference this change.